apps/parse-linter-output/main.go

```diff
@@ -28,7 +28,7 @@ func main() {
 	}
 
 	if *ibmPtr {
-		lint.LintIBM(args[0])
+		lint.LintOpenapiValidator(args[0])
```

Comment:
Can we call this "lintOpenAPIValidator" ? That would be more consistent with OpenAPI branding.